### ISSUE TITLE
Additional Site: Perranporth and Slapton Sands

### ISSUE THREAD

**Issue by HelenD12** [kvos/SDS_Benchmark#12]
I am currently working on an additional site: Perranporth and Slapton Sands and I will post the results once the work is complete.

I am unsure where to get the data from to create the tides.csv and waves_ERA5.csv files? Are there any instructions on how to do this?

I have looked at the CoastSat.slope example and followed the instructions to install the FES2014.

**Comment by ddbuscombe** [comment]
This appears to be a duplicate of https://github.com/SatelliteShorelines/coastsat_package/issues/11
